layers/+tools/org-download/packages-config.el

```diff
@@ -0,0 +1,26 @@
+;;; packages-config.el --- org-download layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Laurent Lejeune <olol85@gmail.com>
+;; URL: https://github.com/krakapwa
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;;; Code:
+(defun org-download/init-org-download()
+  (use-package org-download 
+    :config
+    ))
```

Comment:
`(use-package <package> :defer t)` is a noop, but `(use-package <package>)` is the same as `(require '<package>)`.
@nixmaniack what you're suggesting will work only if org-download defines proper autoloads.